dll/apisets/api-ms-win-crt-private-l1-1-0.spec

```
@@ -40,48 +43,50 @@
 @ stub __current_exception_context
 @ stub __dcrt_get_wide_environment_from_os
 @ stub __dcrt_initial_narrow_environment
-@ stub __intrinsic_abnormal_termination
+@ stub -arch=i386 __intrinsic_abnormal_termination
 @ stub __intrinsic_setjmp
 @ stub __intrinsic_setjmpex
 @ stub __processing_throw
 @ stub __report_gsfailure
 @ stub __std_exception_copy
 @ stub __std_exception_destroy
+@ stub -arch=x86_65 __std_terminate
```

65-bits yay! :partying_face: 
```suggestion
@ stub -arch=x86_64 __std_terminate
```